cpp/bin/torchtrtc/main.cpp

```diff
@@ -147,6 +173,18 @@ int main(int argc, char** argv) {
       "save_engine",
       "Instead of compiling a full a TorchScript program, save the created engine to the path specified as the output path",
       {"save-engine"});
+  args::ValueFlagList<std::string> custom_torch_ops(
+      parser,
+      "custom-torch-ops",
+      "(repeatable) Shared object/DLL containing custom torch operator",
```

Comment:
it should probably be `operators` not `operator`

Reply:
It would make sense to add the instruction for sample to run instead of providing a plugin implementation and then loading it as a plugin in CLI.